### PR TITLE
Forward livechat rooms

### PR DESCRIPTION
**Pull request by sampaiodiego**
@RocketChat/core 

Adds a new button to _Visitor Info_ tab:
image

You can forward either forward to a department or to a specific user:
image

### CODE REVIEW

**Comment by rodrigok** [comment]
LGTM

**Comment by geekgonecrazy** [comment]
Nice :D that could be useful 👍

File: packages/rocketchat-livechat/client/views/app/tabbar/visitorForward.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		return Template.instance().visitor.get();
+	},
+	hasDepartments() {
+		return LivechatDepartment.find({ enabled: true }).count() > 0;
+	},
+	departments() {
+		return LivechatDepartment.find({ enabled: true });
+	},
+	agents() {
+		return AgentUsers.find({ _id: { $ne: Meteor.userId() } }, { sort: { name: 1, username: 1 } });
```
Comment (marceloschmidt) [inline packages/rocketchat-livechat/client/views/app/tabbar/visitorForward.js]: Should we find only `online` agents? Or is this already covered?

Comment (sampaiodiego) [inline packages/rocketchat-livechat/client/views/app/tabbar/visitorForward.js]: I thought about that, but if the other agent is `invisible` (or unavailable for livechat) but is on your side? I thought you'd like to transfer to him anyway =)

I thought then add the online agents first.. what you think?

**Pull request merged**